__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require("../app.js");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+const db  = require("../db/connection")
+
+afterAll(() => db.end())
+
+beforeEach(() => seed(testData))
+
+describe('GET /api/topics', () => { 
+  test('returns 200 and a list of topics', () => { 
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => { 
+        expect(topics).toEqual([
+          {
```

Comment:
This test could be more flexible by checking the shape of the objects inside a forEach. Be aware that you also need to check the length of the array if you go down this path.